lib/functions/versions.bash

```diff
@@ -165,7 +165,7 @@ latest_command() {
     fi
   fi
 
-  printf "%s\n" "$versions"
+  printf "%s %s\n" "$plugin_name" "$versions"
```

Comment:
this is a breaking change

Reply:
Yes, if folks are using the output of `foo=$(asdf latest bar)` in their scripts, that would break.  I'd be happy to add a conditional, with a new configurable option in `~/.asdfrc` like, say, `asdf_latest_include_plugin_name = yes` which defaults to the old behaviour, unless someone opts in?

(of course with a nicer name than `asdf_latest_include_plugin_name` 😅)